ondine/jse-win15-1.md

```diff
@@ -0,0 +1,216 @@
+### Ondine Gallatin
+
+### Homework #1
+(Due Monday 1/12)
+
+---
+
+**1)** (_Difficulty: mixed_)
+Predict the output of each of the following expressions, and explain the rules which determine it.
+If the output depends on the value of x, identify the conditions when the expression will output true.  Assume the cases are independent, and x is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1`
+
+True. The string gets converted to a number and 1 == 1.
+
+**b)** `"1" === 1`
+
+Always false. The operator is strict equals - the operands must be of the same data type as well as value.
+
+**c)** `x == 'x'`
+
+False. Any x value does not equal x that is a string. True only if x is assigned the value 'x'. (x = 'x')
+
+**d)** `x == (x+'')`
+
+Always true. The value of x is the same as x plus an empty string.
+
+**e)** `'' == ' '`
+
+Always false. An empty string is not the same as a string that contains a spaceband.
+
+**f)** `x = true`
+
+True. Boolean value.
+
+**g)** `var x; x == 'undefined'`
+
+False. The value of x is not been declared, so its value is undefined. The type of x is "undefined".
```

Comment:
When var x is declared, it is given an initial value of undefined; the only time this would be false is if var x had previously been given a value.